guides/v2.2/frontend-dev-guide/themes/js-bundling.md

```diff
@@ -1 +1,135 @@
+    <item type="file">Lib::mage/zoom.js</item>
+    <item type="file">Lib::mage/translate-inline-vde.js</item>
+    <item type="file">Lib::mage/requirejs/mixins.js</item>
+    <item type="file">Lib::mage/requirejs/static.js</item>
+    <item type="file">Magento_Customer::js/zxcvbn.js</item>
+    <item type="file">Magento_Catalog::js/zoom.js</item>
+    <item type="file">Magento_Ui::js/lib/step-wizard.js</item>
+    <item type="file">Magento_Ui::js/form/element/ui-select.js</item>
+    <item type="file">Magento_Ui::js/form/element/file-uploader.js</item>
+    <item type="file">Magento_Ui::js/form/components/insert.js</item>
+    <item type="file">Magento_Ui::js/form/components/insert-listing.js</item>
+    <item type="directory">Magento_Ui::js/timeline</item>
+    <item type="directory">Magento_Ui::js/grid</item>
+    <item type="directory">Magento_Ui::js/dynamic-rows</item>
+    <item type="directory">Magento_Ui::templates/timeline</item>
+    <item type="directory">Magento_Ui::templates/grid</item>
+    <item type="directory">Magento_Ui::templates/dynamic-rows</item>
+    <item type="directory">Magento_Swagger::swagger-ui</item>
+    <item type="directory">Lib::modernizr</item>
+    <item type="directory">Lib::tiny_mce</item>
+    <item type="directory">Lib::varien</item>
+    <item type="directory">Lib::jquery/editableMultiselect</item>
+    <item type="directory">Lib::jquery/jstree</item>
+    <item type="directory">Lib::jquery/fileUploader</item>
+    <item type="directory">Lib::css</item>
+    <item type="directory">Lib::lib</item>
+    <item type="directory">Lib::extjs</item>
+    <item type="directory">Lib::prototype</item>
+    <item type="directory">Lib::scriptaculous</item>
+    <item type="directory">Lib::less</item>
+    <item type="directory">Lib::mage/adminhtml</item>
+    <item type="directory">Lib::mage/backend</item>
+</exclude>
+{% endhighlight %}
+
+{% endcollapsible %}
+
+### Setting bundle file size
+
+The `bundle_size` entry controls the file size of the generated bundles.
+Setting this to a large number will reduce the number of bundles generated, but the file sizes will be big.
+A small number will increase the number of bundles generated, but the file sizes will be smaller.
+
+You want to balance the number of files to download on a page with the size of the data.
+A single connection should be around 100kb.
+
+## Fine tuning your theme
+
+There are many ways to tune your theme's `etc/view.xml` file.  
+
+For example, Magento's Luma theme is configured to work well for all pages, but 
+by adding or removing items inside `exclude`, you can maximize browser performance for home, catalog, or product pages.
+
+Follow these steps to help you identify which JavaScript files to bundle for your theme:
+
+1. Create a blank page with the layouts you would like to tune.
+2. Compare the JavaScript files loaded in the pages with the JavaScript files in Magento.
+3. Use the results of that comparison to build your exclude list.
```

Comment:
This section seems a bit unclear to me. I'm wondering if it is re-stating some of the things mentioned earlier? Perhaps part of the issue is that it isn't clear here that files are included automatically, and only files that are manually excluded are not. I like your approach of creating a minimal page for testing!